### ISSUE TITLE
[warning] document.registerElement is deprecated and will be removed in M73 around March 2019 ->location   vendor/nova/vendor.js

### ISSUE THREAD

**Issue by audetcameron** [laravel/nova-issues#974]
Seeing this message in Chrome and wondering if it will be an issue in the future for a production website.

```
[Deprecation] document.registerElement is deprecated and will be removed in M73, around March 2019. Please use window.customElements.define instead. See https://www.chromestatus.com/features/4642138092470272 for more details.
r.registerElement @ vendor.js?id=52886433f04b87b2c184:1
```
![image](https://user-images.githubusercontent.com/6738453/48437875-8c2d9e80-e750-11e8-9b40-43e4bd93fd40.png)

**Comment by davidhemphill** [comment]
This has been fixed and will be included in an upcoming release.

**Comment by thunder87** [comment]
Fixed in 1.1.8 - had to run "php artisan nova:publish" to get it to work 👍
